apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2022-09-13T17:21:37Z"
-  build_hash: 2944c8772f216656d84ee02d392eaca501274c1e
-  go_version: go1.18.3
-  version: v0.20.1
-api_directory_checksum: 68de1519a0eba867c9b7e9a81ff333e524e05948
+  build_date: "2022-12-13T22:56:41Z"
+  build_hash: e661ce95afc39b380653ca655503daebf1e1831b
+  go_version: go1.18.1
+  version: v0.19.3-26-ge661ce9
```

Comment:
Shouldn't this be `v0.21.0`?

Reply:
Need to `git fetch --tags` in your `code-generator` repository 😄

Reply:
My local code-generator was completely outdated. fixed.